var/spack/repos/builtin/packages/compadre/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Compadre(CMakePackage):
+    """The Compadre Toolkit provides a performance portable solution for the
+    parallel evaluation of computationally dense kernels. The toolkit
+    specifically targets the Generalized Moving Least Squares (GMLS) approach,
+    which requires the inversion of small dense matrices. The result is a set
+    of weights that provide the information needed for remap or entries that
+    constitute the rows of some globally sparse matrix.
+    """
+
+    homepage = 'https://github.com/SNLComputation/compadre'
+    git      = 'https://github.com/SNLComputation/compadre.git'
+    url      = 'https://github.com/SNLComputation/compadre/archive/v1.3.0.tar.gz'
+    maintainers = ['kuberry']
+
+    version('1.3.3',  branch='master')
```

Comment:
Can we match the branch name:
```suggestion
    version('master',  branch='master')
```
?

Reply:
Hi @alalazo. The reason I thought to do this was because if we made the suggested change and someone types `spack install compadre` they will get version `1.3.0` unless they are very specific to ask for compadre@master. Am I mistaken? I'm kind of new to spack.

I would prefer they get the version on master by default without having to say compadre@???.

Is there an alternate way to make sure they are served the version on master by default?

Reply:
I think you can still add a keyword argument `preferred=True` to a version called `master` and have that as the preferred version.

Reply:
I made the suggested change. Thanks for the advice!